stdlib/SparseArrays/src/linalg.jl

```diff
@@ -206,7 +206,9 @@ end
 # Frobenius dot/inner product: trace(A'B)
 function dot(A::SparseMatrixCSC{T1,S1},B::SparseMatrixCSC{T2,S2}) where {T1,T2,S1,S2}
     m, n = size(A)
-    size(B) == (m,n) || throw(DimensionMismatch("matrices must have the same dimensions"))
+    if size(B) != (m,n)
```

Comment:
Is `axes` purposely not used here for some reason?

Reply:
I thought `SparseMatrixCSC` uses definitely classical indexing as in line 208. Thus, I've just used `size` instead of `axes`. Should I change that?